copilot/fastapi/tests/test_argo_cd_api.py

```
@@ -0,0 +1,80 @@
+        from app.argo_cd_api import get_argo_cd_token
+        from fastapi import HTTPException
+        import asyncio
+        
+        # Execute the function and verify it raises HTTPException
+        with pytest.raises(HTTPException) as excinfo:
+            asyncio.run(get_argo_cd_token())
+        
+        assert excinfo.value.status_code == 401
+        assert "Authentication Failed" in excinfo.value.detail
+
+def test_argo_cd_proxy(test_client, mock_argo_cd_token):
```

Ruff (0.8.2)

Missing return type annotation for public function `test_argo_cd_proxy`

Add return type annotation: `None`

(ANN201)